webauthn-rs-proto/src/options.rs

```diff
@@ -73,10 +74,11 @@ pub struct User {
     /// must NOT contain personally identifying information, as this value can NEVER
     /// be changed. If in doubt, use a UUID.
     pub id: Base64UrlSafeData,
-    /// The users preferred name for display.
+    /// A detailed name for the account, such as an email address.
```

Comment:
Could be worth commenting that these values can change and should never be used as a primary key.